web/src/components/app.tsx

```diff
@@ -240,10 +240,12 @@ let LocalizedPage: any = class extends React.Component<
                 .reverse()
                 .map(notification =>
                   notification.kind == 'pill' ? (
-                    <NotificationPill
-                      key={notification.id}
-                      {...{ notification }}
-                    />
+                    notification.type !== 'achievement' && (
+                      <NotificationPill
+                        key={notification.id}
+                        {...{ notification }}
+                      />
+                    )
```

Comment:
`Achievement` notification will only show up in the challenge dashboard.

Reply:
I think we want to show achievements everywhere, not just the dashboard!

Reply:
I think catherine means the type notifications in challenge dashboard is `Achievement `notification. Notifications in other board, is other kind of notifications, since they are different UIs.